homeassistant/components/zwave/__init__.py

```diff
@@ -830,6 +830,25 @@ def __init__(self, values, domain):
 
     def network_value_changed(self, value):
         """Handle a value change on the network."""
+        if (value.command_class == const.COMMAND_CLASS_CENTRAL_SCENE and
+                value.node.node_id == self.node.node_id and
+                self.hass is not None):
+
+            scene_id = int(str(value.index) + str(value.data))
+
+            _LOGGER.info(
+                "Central Scene Triggered: Scene %d Key %d "
+                "(activating scene_id=%d for entity_id=%s)",
+                value.index, value.data, scene_id, self.entity_id)
+
+            self.hass.bus.fire(const.EVENT_SCENE_ACTIVATED, {
+                ATTR_ENTITY_ID:       self.entity_id,
+                const.ATTR_NODE_ID:   self.node.node_id,
+                const.ATTR_SCENE_ID:  scene_id
```

Comment:
Just change this to
```python
const.ATTR_VALUE_INDEX: value.index,
const.ATTR_VALUE_DATA: value.data,
```

[In const.py](https://github.com/home-assistant/home-assistant/blob/bd039b8c5332e9225ef2bd8291c9fd12309bebd9/homeassistant/components/zwave/const.py#L17), you'll need to define `ATTR_VALUE_DATA`
```python
ATTR_VALUE_DATA = "value_data"
```